kolibri/plugins/facility/assets/src/views/SelectionBottomBar.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+
+  <BottomAppBar>
+    <span class="message">{{ selectedMessage }}</span>
+    <KButton
+      :disabled="$attrs.disabled || buttonsDisabled"
+      :text="coreString('confirmAction')"
+      :primary="true"
+      @click="$emit('click-confirm')"
+    />
+  </BottomAppBar>
+
+</template>
+
+
+<script>
+
+  import commonCoreStrings from 'kolibri.coreVue.mixins.commonCoreStrings';
+  import BottomAppBar from 'kolibri.coreVue.components.BottomAppBar';
+  import responsiveWindowMixin from 'kolibri.coreVue.mixins.responsiveWindowMixin';
```

Comment:
I don't think anything in the component is using this mixin so this import can be removed

```suggestion
```